packages/react/src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -72,47 +30,30 @@ const getIconSize = (size: SizeValue, sizeModifier: IconSizeModifier): number =>
     },
   }
 
-  return modifiedSizes[size] && modifiedSizes[size][sizeModifier]
+  return modifiedSizes[size] && pxToRem(modifiedSizes[size][sizeModifier])
 }
 
 const getIconColor = (variables, colors) => {
   return _.get(colors, 'foreground', variables.color || 'currentColor')
 }
 
 const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
-  root: ({
-    props: { disabled, name, size, bordered, circular, color, xSpacing, rotate },
-    variables: v,
-    theme,
-  }): ICSSInJSStyle => {
-    const colors = v.colorScheme[color]
-    const iconSpec = theme.icons[name]
-    const isFontBased = name && (!iconSpec || !iconSpec.isSvg)
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const colors = v.colorScheme[p.color]
 
     return {
-      backgroundColor: v.backgroundColor,
-      boxSizing: isFontBased ? 'content-box' : 'border-box',
-
-      // overriding the base theme default transformation as in teams theme the svg/svgFlippingInRtl slots are used for this
-      ...(!isFontBased && {
-        transform: 'unset',
-      }),
-
-      ...(isFontBased && {
-        ...getFontStyles(getIconSize(size, v.sizeModifier), name),
-        fontWeight: 900, // required for the fontAwesome to render
-        color: getIconColor(v, colors),
-        transform: `rotate(${rotate}deg)`,
-        ...(disabled && {
-          color: v.disabledColor,
-        }),
-      }),
-
       // overriding base theme border handling
-      ...((bordered || v.borderColor) &&
+      ...((p.bordered || v.borderColor) &&
         getBorderedStyles(v.borderColor || getIconColor(v, colors))),
     }
   },
+  svgRoot: ({ props: p, variables: v }): ICSSInJSStyle => {
+    return {
+      display: 'inline-block',
+      backgroundColor: v.backgroundColor,
+      boxSizing: 'border-box',
```

Comment:
No need for specifying boxSizing: 'border-box'. Teams theme users only svg icons, and there is no prev definition of 'content-box' (it is for font icons in base theme, not svg)